test/e2e_node/topology_manager_test.go

```diff
@@ -199,8 +199,6 @@ func configureTopologyManagerInKubelet(oldCfg *kubeletconfig.KubeletConfiguratio
 		newCfg.FeatureGates = make(map[string]bool)
 	}
 
-	newCfg.FeatureGates["TopologyManager"] = true
```

Comment:
I think tests do not need the tags `[Feature:TopologyManager][NodeFeature:TopologyManager]` any longer

Reply:
A tag such as this can be useful if someone want to run just topology manager node e2e tests e.g. for debugging purposes. I will remove `[Feature:TopologyManager]` and we can keep just `[NodeFeature:TopologyManager]`.